README.md

```diff
@@ -79,6 +79,59 @@ Run the demo in your browser:
 rpm run start
 ```
 
+## Gosling API
+
+To use the Gosling API, we need to create a reference to the GoslingComponent.
```

Comment:
Just added a link that might be helpful for people who are not familiar with React

```suggestion
To use the Gosling API, you need to create a [Ref](https://reactjs.org/docs/refs-and-the-dom.html) that stores a reference to the GoslingComponent.
```